src/frontend/js/widgets/Dashboard/components/TeacherDashboardProfileSidebar/components/OrganizationLinks/_styles.scss

```diff
@@ -18,13 +18,14 @@
     }
 
     &__link {
-      flex: 0 0 32%;
-      display: flex;
-      border: thin solid r-theme-val(organization-thumb, border-color);
+      align-items: center;
       border-radius: rem-calc(8px);
-      overflow: hidden;
+      border: thin solid r-theme-val(organization-thumb, border-color);
+      display: flex;
+      flex: 0 0 32%;
       justify-content: center;
-      align-items: center;
+      max-width: 64px;
```

Comment:
rem?

Reply:
Finally I applied this style only on viewport smaller than `lg` breakpoint